bookwyrm/views/search.py

```diff
@@ -123,3 +125,35 @@ def list_search(query, viewer, *_):
         )
         .order_by("-similarity")
     ), None
+
+
+def isbn_check(query):
+    """isbn10 or isbn13 check, if so remove separators"""
+    if query:
+        su_num = re.sub(r"(?<=\d)\D(?=\d|[xX])", "", query)
```

Comment:
I'm not sure what `su_num` means - is there a more verbose way to name it so it's more clear? Alternatively, a comment on the line aboveexplaining it would help me out